src/editors/content/container/ContentContainer.tsx

```diff
@@ -385,3 +386,36 @@ export class ContentContainer
   }
 
 }
+
+interface OwnProps extends AbstractContentEditorProps<ContentElements> {
+  className?: string;
+  hideContentLabel?: boolean | string[];
+  disableContentSelection?: boolean | string[];
+  bindProperties?: (element: ContentElement) => BoundProperty[];
+  activeContentGuid: string;
+  hideSingleDecorator?: boolean;
+  hideAllDecorators?: boolean;
+  layout?: Layout;
+  selectedEntity?: Maybe<string>;
+  overrideRemove?: (model: ContentElements, childModel: Object) => boolean;
+}
+
+interface StateProps { }
+
+interface DispatchProps {
+  onUpdateEditor: (editor) => void;
+}
+
+const mapStateToProps = (state, ownProps): StateProps => {
+  return {};
+};
+
+const mapDispatchToProps = (dispatch, ownProps): DispatchProps => {
+  return {
+    onUpdateEditor: editor => dispatch(updateEditor(editor)),
+  };
+};
+
+const controller = connect<StateProps, DispatchProps, OwnProps>
+  (mapStateToProps, mapDispatchToProps)(ContentContainer);
+export { controller as ContentContainer };
```

Comment:
I didn't separate the controller into a separate file because `ContentContainer` is used in like 100 places